.setup/bin/setup_sample_courses.py

```diff
@@ -192,6 +192,13 @@ def main():
     os.system("crontab -u hwcron /tmp/hwcron_cron_backup.txt")
     os.system("rm /tmp/hwcron_cron_backup.txt")
 
+def generate_random_user_id(length):
+    id_base = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+    anon_id = ""
+    for i in range(length):
+        anon_id += random.choice(id_base)
+    return anon_id
```

Comment:
could replace the above four lines with `return ''.join(random.choice(id_base) for _ in range(length))`